src/translations/locales/en/common.json

```
@@ -1259,7 +1264,8 @@
     "exchangeAllowanceFailed": "We are sorry. Token allowance could not be set. Please try again.",
     "contactUpdated": "Contact updated",
     "contactDeleted": "Contact deleted",
-    "contactAdded": "New contact added"
+    "contactAdded": "New contact added",
+    "smartWalletActivationUnavailable": "Activation is temporarily unavailable. Please try again latter"
```

This string had typo before:
```suggestion
    "smartWalletActivationUnavailable": "Activation is temporarily unavailable. Please try again later"
```